connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -43,6 +43,10 @@
             + "than one, though, in case a server is down).";
     public static final String BOOTSTRAP_SERVERS_DEFAULT = "localhost:9092";
 
+    public static final String CONNECT_PREFIX_CONFIG = "connect.prefix";
+    public static final String CONNECT_PREFIX_DOC = "Prefix for sink task consumer group.";
```

Comment:
Since the motivation for using this probably isn't too obvious to someone who isn't familiar with the implementation details, it'd probably be worth expanding this explanation a bit to say when you would use it.